www/style/codecheckerviewer.css

```diff
@@ -938,3 +963,95 @@ span[class*="severity-"] {
   content: "\e01d";
 }
 
+/* Run history */
+.history-group {
+  margin-bottom: 10px;
+  border: 1px solid #e1e4e8;
+  border-radius: 5px;
+}
+
+.history-group .header {
+  font-size: 1.5em;
+  font-weight: bold;
+  color: #000;
+  font-family: Times, serif;
```

Comment:
Is this style here intentional? We don't really use _serif_ fonts in the GUI. Looks awkward.